exercises/exercise_000_sudoku_solver_initial_state/src/main/scala/org/lunatechlabs/dotty/SudokuSolverMain.scala

```diff
@@ -42,11 +43,12 @@ object Main {
 object SudokuSolverMain {
 
   def main(args: Array[String]): Unit = {
+    Kamon.init()
 
     val system = ActorSystem[NotUsed](Main(), "sudoku-solver-system")
 
     println(s"${GREEN}Hit RETURN to stop solver${RESET}")
```

Comment:
`run / fork := true` is required for Kamon instrumentation to work (if the application doesn't run in a forked JVM then the instrumentation agent can't be attached so no metrics are sent anywhere) 

BUT, this stops the application blocking on `StdIn.readline()` - hence the need to remove the two lines below (otherwise it just falls through to `system.terminate()`) - not sure why, seems to be a known thing.

We should remove them completely rather than just commenting out, and probably remove the `println` as well as it's not necessary either.

Reply:
> We should remove them completely rather than just commenting out, and probably remove the println as well as it's not necessary either.

Yes, please remove it as needed